buildscripts/docker/docker_release/env.yaml

```diff
@@ -0,0 +1,11 @@
+name: base
+channels:
+  - conda-forge
+  - bodo.ai
+dependencies:
+  - python=3.12
+  - bodo
+  - bodosql
```

Comment:
Should be include everything, include the Iceberg connector and the Azure SAS token

Reply:
Also, should we fix the version somehow to be safe. Like lets say we accidentally publish one release after another, then the Docker build will get a later version of bodo

Reply:
so this is part of the release process so it should always get water was released.
I don't think we should fix version.

Reply:
No thats not necessarily true. Take my example.

We release 2025.1.1.
Linux Conda 2025.1.1 is built.
We notice a problem and then release 2025.1.2
Linux Conda 2025.1.2 is built.
The docker image for 2025.1.1 runs, it will pick up Bodo 2025.1.2.

Now ofc if we're good at keeping up with the CIs and make sure to cancel 2025.1.1 before its too late, then this is fine. But I don't know if we always are. This can be a followup fix though